addons/interaction/functions/fnc_getGlassDoor.sqf

```diff
@@ -0,0 +1,67 @@
+/*
+ * Author: Phyma
+ * Find glass door.
+ *
+ * Arguments:
+ * 0: Distance <NUMBER>
+ * 1: House <OBJECT>
+ * 2: Door name <STRING>
+ *
+ * Return Value: 
+ * 0: Door Name <STRING>
+ *
+ * Example:
+ * [player, target] call ace_interaction_fnc_getGlassDoor
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params ["_distance", "_house", "_door"]; 
+
+private ["_animName", "_splitStr", "_newString", "_objDist", "_animate"];
```

Comment:
delete this
`_objDist` and `_animate` are already set to private in their scopes.